include/librealsense2/h/rs_frame.h

```diff
@@ -62,9 +62,9 @@ typedef enum rs2_frame_metadata_value
     RS2_FRAME_METADATA_FRAME_LED_POWER                      , /**< Led power value 0-360. */
     RS2_FRAME_METADATA_RAW_FRAME_SIZE                       , /**< The number of transmitted payload bytes, not including metadata */
     RS2_FRAME_METADATA_GPIO_INPUT_DATA                      , /**< GPIO input data */
-    RS2_FRAME_METADATA_SUBPRESET_ID                         , /**< sub-preset id */
-    RS2_FRAME_METADATA_SUBPRESET_SEQUENCE_ID                , /**< sub-preset sequence id */
-    RS2_FRAME_METADATA_SUBPRESET_SEQUENCE_SIZE              , /**< sub-preset sequence size */
+    RS2_FRAME_METADATA_SEQUENCE_NAME                         , /**< sub-preset id */
```

Comment:
Please extract the renaming into a separate PR

Reply:
done